src/constructs/cloudwatch/ec2-alarms.ts

```diff
@@ -5,18 +5,9 @@ import type { GuStack } from "../core";
 import { AppIdentity } from "../core";
 import type { GuApplicationLoadBalancer, GuApplicationTargetGroup } from "../loadbalancing";
 import { GuAlarm } from "./alarm";
-import type { GuAlarmProps } from "./alarm";
+import type { GuAlarmProps, Http5xxAlarmProps } from "./alarm";
 
-export interface Http5xxAlarmProps
```

Comment:
This interface is now shared, see https://github.com/guardian/cdk/pull/1250/files#r867990801